openapi.yaml

```
@@ -372,15 +375,13 @@ tags:
       ```
       ### Example API call for GWA Intraday LTC:
       ```
-      curl https://api.bravenewcoin.com/v3/index-ticker?indexType=GWA&indexId={LTC_Asset_Id} -H 'Authorization: Bearer test_SiHrnL5NKsea0G2W4LHd' -G
+      curl https://api.bravenewcoin.com/v3/index-ticker?timestamp=2019-09-25T00:00:00.000Z&indexType=GWA&indexId={LTC_Asset_Id} -H 'Authorization: Bearer test_SiHrnL5NKsea0G2W4LHd' -G
```

```suggestion
      curl https://api.bravenewcoin.com/v3/index-ticker?timestamp=2020-03-17T00:00:00.000Z&indexType=GWA&indexId=220cbd0c-e466-4a75-9833-755307f872f3 -H 'Authorization: Bearer test_SiHrnL5NKsea0G2W4LHd' -G
```

220cbd0c-e466-4a75-9833-755307f872f3 is the asset ID for Litecoin.  I wonder if we should provide examples that will work?